app/views/publishers/vacancies/job_applications/_messages.html.slim

```diff
@@ -4,30 +4,22 @@
       = govuk_button_link_to t(".print_page"), "#", class: "govuk-button--secondary", "data-controller": "utils", "data-action": "click->utils#print" unless params[:show_form] == "true"
       = govuk_button_link_to t(".send_message_to_candidate"), url_for(params.to_unsafe_h.merge(show_form: "true")), class: "govuk-button--secondary govuk-!-margin-left-3" unless params[:show_form] == "true"
 
-    - if params[:show_form] == "true"
-      div class="govuk-!-margin-top-6"
-        = govuk_details summary_text: t(".how_will_message_be_sent") do
-          = t(".message_sent_description")
```

Comment:
did we mean to change the behaviour here? This boolean used to be required in order for the form to be shown